test/lib/next-modes/base.ts

```diff
@@ -550,6 +550,10 @@ export class NextInstance {
     return fetchViaHTTP(this.url, pathname, null, opts)
   }
 
+  public async webFetch(pathname: string, opts?: Parameters<typeof fetch>[1]) {
```

Comment:
Added this as a temporary stopgap. Ideally we switch the fetch method on `NextInstance` to use the web fetch but that's for another PR.